### PR TITLE
PLATML-5094 - Updated runtime version in Dockerfile

### PR DESCRIPTION
**Pull request by NemanjaL94**

### CODE REVIEW

**Review by tumulurik — APPROVED** [review]
LGTM

File: recipes/R/Retail - GradientBoosting/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-r:0.17.1
+FROM adobe/acp-dsw-ml-runtime-r:0.17.2
```
Comment (rjrussell77) [inline recipes/R/Retail - GradientBoosting/Dockerfile:18]: Thanks, @NemanjaL94

**Review by rjrussell77 — APPROVED** [review]
LGTM

File: recipes/R/Retail - GradientBoosting/Dockerfile (hunk 1 above)
Comment (NemanjaL94) [reply recipes/R/Retail - GradientBoosting/Dockerfile:18]: Anytime @rjrussell77

**Pull request merged**